src/lj_jitlog.c

```diff
@@ -208,6 +208,17 @@ static void jitlog_tracestart(JITLogState *context, GCtrace *T)
   context->lastpc = proto_bcpos(J->pt, J->pc);
 }
 
+static int isstitched(JITLogState *context, GCtrace *T)
+{
+  jit_State *J = G2J(context->g);
+  if (J->parent == 0) {
+    BCOp op = bc_op(T->startins);
+    /* The parent trace rewrites the stack so this trace is started after a the untraceable call */
```

Comment:
"a the"

Reply:
Thats actually a copy pasted comment from LuaJIT itself.

Reply:
Then feel free to fix both instances ;)